src/app/ui/surface/NestedTabs.module.css

```
@@ -0,0 +1,363 @@
+  --nested-tabs-scroll-fade: 1.35rem;
+
+  min-height: 0;
+  padding: 0.4rem 0.25rem;
+  overflow-x: hidden;
+  overflow-y: auto;
+  scrollbar-color: transparent transparent;
+  scrollbar-width: thin;
+}
+
+.levelItems[data-scrolling] {
+  scrollbar-color: color-mix(in srgb, currentColor 38%, transparent) transparent;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)